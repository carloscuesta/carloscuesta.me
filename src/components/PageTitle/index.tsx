type Props = {
  title: string
}

const PageTitle = (props: Props) => (
  <h1 className="my-6 text-xl font-bold">{props.title}</h1>
)

export default PageTitle
