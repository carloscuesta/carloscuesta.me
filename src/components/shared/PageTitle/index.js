// @flow
import React, { type Element } from 'react'

import styles from './styles.module.css'

type Props = {
  title: string
}

const PageTitle = (props: Props): Element<'h1'> => (
  <h1 className={styles.title}>
    {props.title}
  </h1>
)

export default PageTitle
