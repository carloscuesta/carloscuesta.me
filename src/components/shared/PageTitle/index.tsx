import styles from './styles.module.css'

type Props = {
  title: string
}

const PageTitle = (props: Props) => (
  <h1 className={styles.title}>
    {props.title}
  </h1>
)

export default PageTitle
