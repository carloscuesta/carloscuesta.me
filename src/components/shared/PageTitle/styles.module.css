.title {
  font-size: 3rem;
  font-weight: 700;
  margin-bottom: 0;
}
