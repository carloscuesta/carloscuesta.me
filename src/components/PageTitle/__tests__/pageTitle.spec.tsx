import { render } from '@testing-library/react'

import PageTitle from '../index'

describe('PageTitle', () => {
  it('should match PageTitle component', () => {
    const { container } = render(<PageTitle title="Some title" />)

    expect(container).toMatchSnapshot()
  })
})
