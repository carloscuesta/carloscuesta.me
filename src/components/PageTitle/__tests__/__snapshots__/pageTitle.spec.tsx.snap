// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle should match PageTitle component 1`] = `
<h1
  className="my-6 text-xl font-bold"
>
  Some title
</h1>
`;
